Automattic/jetpack
# Search: show search submenu for standalone with Jetpack plugin and site connected

Fixes #24312 

#### Changes proposed in this Pull Request:
- Added compatibility files for search plugin: show search submenu if Search plugin exists with Jetpack plugin and site connected
- Change loading process for chances of hooking before initialization

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Create a JN site with Search plugin and Jetpack of the branch `update/show-search-submenu-for-standalone`
- Ensure Search submenu is NOT shown
- Connect the site
- Ensure Search submenu is added and we could access upsell page by clicking it
- Deactivate Search plugin
- Ensure Search submenu is removed
